zeebe/test-util/src/main/java/io/camunda/zeebe/test/util/record/CompactRecordLogger.java

```diff
@@ -88,6 +89,7 @@ public class CompactRecordLogger {
           entry("PROCESS_INSTANCE_MODIFICATION", "MOD"),
           entry("PROCESS_INSTANCE", "PI"),
           entry("PROCESS", "PROC"),
+          entry("AD_HOC_SUB_PROCESS_ACTIVITY_ACTIVATION", "ADHOC_ACT"),
```

Comment:
🔧  I didn't follow the logic, but it seems this breaks the logic of the second column in the log having a very short identifier (e.g. `PI`). Can we double-check we set this up as the other logs?

<img width="1154" alt="image" src="https://github.com/user-attachments/assets/3be16269-feec-4b14-b896-c66fdf636ba3" />

Reply:
Good catch! I wonder why this is happening 🤔 Because as far as I was aware, the mapping of the entry is enough. Because the first parameter is the name of the enum variant, and the second parameter is the abbreviation. But I will double check this. 

Also, maybe I can change the abbreviated for to `ACT`. Wdyt?

Reply:
Not sure if `ACT` is too generic, looking at the others the currently used `ADHOC_ACT` might be a good fit.  Maybe we could abbreviate to `AH_ACT`?

Reply:
`AH_ACT` seems like a good abbreviation. Will go for that!